renovate.json

```diff
@@ -3,5 +3,12 @@
   "extends": ["github>konflux-ci/mintmaker//config/renovate/renovate.json"],
   "timezone": "America/New_York",
   "schedule": ["before 10pm"],
-  "enabledManagers": ["asdf", "tekton"]
+  "enabledManagers": ["asdf", "tekton", "github-actions"],
+  "github-actions": {
+    "enabled": true,
+    "schedule": ["before 10pm"],
+    "separateMinorPatch": true,
```

Comment:
What does this one mean?

Reply:
it should break the PRs up. https://github.com/renovatebot/renovate/blob/main/docs/usage/faq.md